src/navigation/BottomTabs/BottomTabNavigator.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {createBottomTabNavigator} from '@react-navigation/bottom-tabs';
+import {PrimaryStackNavigator} from '../PrimaryStack';
+import {BottomTabRoute} from './BottomTabNavigator.types';
+import {CustomIcon, CustomIconNames} from '../../components';
+import ProfileScreen from '../../screens/ProfileScreen/ProfileScreen';
+
+const BottomTabs = createBottomTabNavigator();
+
+const BottomTabNavigator = () => {
+  const tabBarIcon = (props: {
+    focused: boolean;
+    color: string;
+    size: number;
+    icon: CustomIconNames;
+  }) => {
+    return (
+      <CustomIcon name={props.icon} size={props.size} color={props.color} />
```

Comment:
Are the icons same on focused as well? If icons are same then maybe we can at least change the color to highlight the selected tab

Reply:
I am Changing the color of icons from red to grey,. Red when active and Grey when inactive.